tests/src/JIT/Methodical/Boxing/xlang/sin_cs.il

```diff
@@ -10,7 +10,7 @@
 }
 .assembly extern legacy library mscorlib { auto }
 .assembly 'sin_il_cs' {}
-.assembly extern sinlib_cs
+.assembly extern sinlib
```

Comment:
Instead of this, please rename sinlib to sinlib_cs.  I realize that requires more changes, but I prefer making this test consistent with itself (i.e. versus sinlib_il) and with the legacy harness (it was called sinlib_cs there).